### PR TITLE
(CDPE-6560) Fixes for release

### PR DESCRIPTION
**Pull request by sean-mckenna**
### Description of proposed changes

- Add npm auth token to publish github action
- Move last artifactory dep to the npm registry
- Fix imports casing that was causing problems with the website storybook build
- Correct CHANGELOG react-components version

### CODE REVIEW

**Comment by sean-mckenna** [comment]
Testing whether the ACCESS_TOKEN secret is actually the npm auth token needed to publish to the registry.

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
       - run: npm run build --if-present
       - run: npm test
 
-  deploy:
+  publish:
```
Comment (mttsltrs) [inline .github/workflows/publish.yml:21]: Maybe rename this to release?

**Review by mttsltrs — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (sean-mckenna) [reply .github/workflows/publish.yml:21]: Just this job or the whole action?

**Comment by sean-mckenna** [comment]
@mttsltrs I think for now il just revert most of the changes in the publish action and commit the other fixes in this PR. Il continue testing the publish action in a separate branch.

**Pull request merged**